x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportStopDatafeedAction.java

```diff
@@ -64,18 +64,18 @@ public TransportStopDatafeedAction(Settings settings, TransportService transport
     }
 
     /**
-     * Resolve the requested datafeeds and add their IDs to one of the list
-     * arguments depending on datafeed state.
+     * Sort the datafeed IDs the their task state and add to one
+     * of the list arguments depending on the state.
      *
      * @param expandedDatafeedIds The expanded set of IDs
      * @param tasks Persistent task meta data
      * @param startedDatafeedIds Started datafeed ids are added to this list
      * @param stoppingDatafeedIds Stopping datafeed ids are added to this list
      */
-    static void resolveDataFeedIds(Set<String> expandedDatafeedIds,
-                                   PersistentTasksCustomMetaData tasks,
-                                   List<String> startedDatafeedIds,
-                                   List<String> stoppingDatafeedIds) {
+    static void sortDatafeedIdsByTaskState(Set<String> expandedDatafeedIds,
```

Comment:
Note we also have to rename the unit tests.

Reply:
done